src/pretix/control/forms/event.py

```diff
@@ -240,6 +240,21 @@ class EventSettingsForm(SettingsForm):
         help_text=_("Require customers to fill in the names of all attendees."),
         required=False
     )
+    attendee_emails_asked = forms.BooleanField(
+        label=_("Ask for attendee e-mails"),
+        help_text=_("Ask for an e-mail address for all tickets which include admission to the event. Important: For "
```

Comment:
I get what you're saying - I think my version was confusing because I was kind of confused about the purpose of this (which adds to my point: maybe the label "Ask for attendee e-mails" itself should be changed, too).